raft/src/test/java/org/apache/kafka/raft/internals/BatchAccumulatorTest.java

```diff
@@ -354,6 +484,7 @@ public void testDrainDoesNotBlockWithConcurrentAppend() throws Exception {
             completedBatch.data.batches().forEach(recordBatch -> {
                 assertEquals(leaderEpoch, recordBatch.partitionLeaderEpoch()); });
         });
+        acc.close();
```

Comment:
Out of curiosity, why do we do this here but not elsewhere in this class?

Reply:
Hmm, not sure how I ended up adding that there